testfiles/5ZoneWaterLoopHeatPump.idf

```diff
@@ -2909,7 +2612,7 @@
     38000,                   !- Nominal Cooling Coil Capacity {W}
     0,                       !- Nominal Time for Condensate Removal to Begin {s}
     0,                       !- Ratio of Initial Moisture Evaporation Rate and Steady State Latent Capacity {dimensionless}
-    3000000,                 !- High Pressure Cutoff {Pa}
+    4000000,                 !- High Pressure Cutoff {Pa}
```

Comment:
Yes.  Followed the lead of another similar example file (don't remember which one now).